### PR TITLE
Add more tips for starting new CoffeeOps meetups

### PR DESCRIPTION
**Pull request by unixorn**
* Add some more tips for people starting a new CoffeeOps meetup

### CODE REVIEW

**Comment by unixorn** [comment]
The failure appears to be that travis won't open an https connection to coffeeops.slack.com, though that works fine in the browser.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ When first starting out, you'll probably have just a few folks (2-3). So, let ev
 2. Availability and accessibility of the chosen venue
    * For example, is it easy to get to for the people you are targeting? Check for steps/elevator availability
    * What is the parking situation like?
+   * Conduct a poll about locations 
 3. Provide a few ice breaker ideas to get folks talking
 4. **Marketing** .. the how you let folks know the event is happening
+   * [meetup.com](https://meetup.com) is a good way to publicize your meetup in the beginning.
+   * Slack. If there's a dev or ops oriented slack for your area, see if they have some sort of #meetup channel you can post announcements in.
+   * [CoffeeOps slack](https:/coffeeops.slack.com) - If there isn't a #cityname channel, make one. Make a point of posting every meeting there so that new members of the slack can find out there's now a CoffeeOps in the area.
```
Comment (iennae) [inline README.md]: missing a / before coffeeops

Comment (unixorn) [reply README.md]: Doh!

Comment (unixorn) [reply README.md]: Fixing, thanks

**Review by iennae — APPROVED** [review]

**Pull request merged**